Project.toml

```diff
@@ -3,7 +3,7 @@ uuid = "c4c1e6a2-6bdd-52e0-b56d-1d4734724d2d"
 version = "1.2.1"
```

Comment:
Guess we need to bump this to `2.0.0` now.

Reply:
I wouldn't call this a breaking change per se. To be safe it could be 2.0 if you'd prefer, but the interface itself hasn't changed, so 1.3 would be justifiable IMO.

Reply:
We at Invenia *might* be doing this somewhere, I doubt anyone else is though. But it would break in the case of some code using, `AWS.jl`, `AWSCore.jl` and `CloudWatchLogs.jl`. Where the user would need to change passing `AWSCore.AWSConfig -> AWS.AWSConfig`.

I think also tagging as `2.0.0` is a bit safer as it won't auto update a bunch of stuff and potentially break it in scenarios like the above.

Reply:
Sure, whatever you think is best.